src/user/authorization/SignUpPage.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Button, FormControl, FormLabel, Input, InputGroup, InputRightElement, Text, useToast } from "@chakra-ui/react";
+import { Heading } from "@chakra-ui/react"
+import React from "react";
+import "./style.css"
+import Creds from "./Creds"
+import authorizationClientInstance from "./api/AuthorizationClientImpl";
+import { Link } from "react-router-dom";
+
+
+export default function SignUpPage() {
+    const [show, setShow] = React.useState(false)
+    const toast = useToast()
+    const toastIdRef = React.useRef()
+
+    const handleClick = () => setShow(!show)
+  
+    const [username, setUsername] = React.useState("")
+    const [password, setPassword] = React.useState("")
+
+    const handleSubmit = () => {
+        let creds: Creds
+        creds = {username: username, password: password}
+    
+        authorizationClientInstance.signUp(creds).then(
+            res => {
+                console.log(res)
+                if(res.status === 201) {
+                    showCreatedAccount()
+                }
+            }
+
+        ).catch(error => {
+            console.log(error)
+            showSigningUpFailure()
+        });
+    }
+
+    const showCreatedAccount = () => {
+        toastIdRef.current = toast({description: "Created account successfully!", status: "success"}) as any
+    }
+
+    const showSigningUpFailure = () => {
+        toastIdRef.current = toast({description: "Error!", status: "error"}) as any
+    }
+  
+    return (
+        <>
+            <Box className="AuthorizationPageBox" rounded="lg">
+                <Box className="AuthorizationPageBoxContent">
+                    <Heading as="h3" size="lg" paddingTop="20px" marginBottom="5px">Sign up</Heading>
+                    <Text fontSize="2xl" marginBottom="25px" className="Logo">service logo</Text>
+                        <FormControl isRequired mt={6}>
+                            <FormLabel textAlign="left" mb="8px"> Username or email</FormLabel>
```

Comment:
There is a space at the front of text. Consider i18n
```suggestion
                            <FormLabel textAlign="left" mb="8px">Username or email</FormLabel>
```